### ISSUE TITLE
sys-kernel/scx-1.0.2: buld failed in scx_lavd

### ISSUE THREAD

**Issue by td5038** [xarblu/xarblu-overlay#508]
The build for scx_lavd failed with some resolution issues. 

```emerge --info``` output:

```
Portage 3.0.65 (python 3.12.5-final-0, default/linux/amd64/23.0/desktop/plasma/systemd, gcc-13, glibc-2.40, 6.10.5-linux-cachyos x86_64)
=================================================================
                         System Settings
=================================================================
System uname: Linux-6.10.5-linux-cachyos-x86_64-AMD_Ryzen_7_5800H_with_Radeon_Graphics-with-glibc2.40
KiB Mem:    15713632 total,    830484 free
KiB Swap:   25165820 total,  24306172 free
Timestamp of repository gentoo: Fri, 16 Aug 2024 23:04:23 +0000
Head commit of repository gentoo: a77bcce2c33ee5adbf058b21fa0e469f57788471

Timestamp of repository CachyOS-kernels: Fri, 16 Aug 2024 17:37:01 +0000
Head commit of repository CachyOS-kernels: eb243ef86c097553cf29a158dce965ef78229272

Timestamp of repository ROKO__: Fri, 16 Aug 2024 17:36:19 +0000
Head commit of repository ROKO__: eeff596c135471b6263d7bcabf7832736fe865f6

Timestamp of repository crossdev: Sun, 20 Jun 2021 21:41:33 +0000
Head commit of repository crossdev: 7c86f8ca262ff27fe4fb1965d8f8fef19f77a3bb

Timestamp of repository dlang: Fri, 16 Aug 2024 17:36:21 +0000
Head commit of repository dlang: 11c266e51e7615df9c91f4d425e6abc6d24edcb3

Timestamp of repository gentoo-unity7: Fri, 16 Aug 2024 17:37:02 +0000
Head commit of repository gentoo-unity7: a40f6c9dd2bd2df55566a166182e44f345f3ff81

Timestamp of repository gentoo-zh: Fri, 16 Aug 2024 17:36:23 +0000
Head commit of repository gentoo-zh: 52a98c08f4fa62c4d36191253b311a88d05c9d00

Timestamp of repository guru: Fri, 16 Aug 2024 17:36:43 +0000
Head commit of repository guru: 14383152d267c68e7d26d9dff961282cdca0e406

Timestamp of repository haskell: Fri, 16 Aug 2024 17:36:23 +0000
Head commit of repository haskell: 20104bfbd89225fd6e85e275776042fadcbc1106

Timestamp of repository kde: Fri, 16 Aug 2024 20:18:58 +0000
Head commit of repository kde: e9ca81facbbe3504533c7914396757a751d5812b

Timestamp of repository lto-overlay: Fri, 16 Aug 2024 17:36:38 +0000
Head commit of repository lto-overlay: 55328d2dee8fc5f86d5bd1b4ee32a66ff173e051

Timestamp of repository mv: Fri, 16 Aug 2024 17:36:25 +0000
Head commit of repository mv: 49effff1b689c32f5f7bd380a3d8a9a5e20cf305

Timestamp of repository steam-overlay: Fri, 16 Aug 2024 17:36:30 +0000
Head commit of repository steam-overlay: bbfa43984c963cd4572d63aec3264de26ad1ab98

Head commit of repository td5038-overlay: a5b5a5f3672af6c383fb9f911b117e4fbd50dca6

Head commit of repository the-anime-team: b33baef43de8d9ac9269f82a239818172c131a41

Timestamp of repository wayland-desktop: Fri, 16 Aug 2024 17:36:47 +0000
Head commit of repository wayland-desktop: 7b27c3773763c09d4e9043e9f90b3f53ad009e7e

Timestamp of repository xarblu-overlay: Fri, 16 Aug 2024 17:36:58 +0000
Head commit of repository xarblu-overlay: cb1d48f622acc3c7fb7a1a0389f1610cb65d436d

sh bash 5.2_p32
ld GNU ld (Gentoo 2.42 p6) 2.42.0
app-misc/pax-utils:        1.3.7::gentoo
app-shells/bash:           5.2_p32::gentoo
dev-build/autoconf:        2.13-r8::gentoo, 2.72-r1::gentoo
dev-build/automake:        1.17-r1::gentoo
dev-build/cmake:           3.30.2::gentoo
dev-build/libtool:         2.4.7-r4::gentoo
dev-build/make:            4.4.1-r1::gentoo
dev-build/meson:           1.5.1::gentoo
dev-java/java-config:      2.3.4::gentoo
dev-lang/perl:             5.40.0::gentoo
dev-lang/python:           3.12.5::gentoo, 3.13.0_rc1_p2::gentoo
dev-lang/rust:             1.80.1::gentoo
dev-lang/rust-bin:         1.80.1::gentoo
sys-apps/baselayout:       2.15::gentoo
sys-apps/sandbox:          2.39::gentoo
sys-apps/systemd:          256.5::gentoo
sys-devel/binutils:        2.42-r2::gentoo, 2.43::gentoo
sys-devel/binutils-config: 5.5.2::gentoo
sys-devel/clang:           17.0.6::gentoo, 18.1.8::gentoo, 19.1.0_rc2::gentoo
sys-devel/gcc:             13.3.1_p20240614::gentoo, 14.2.0::gentoo
sys-devel/gcc-config:      2.11::gentoo
sys-devel/lld:             18.1.8::gentoo, 19.1.0_rc2::gentoo
sys-devel/llvm:            17.0.6-r2::gentoo, 18.1.8-r4::gentoo, 19.1.0_rc2::gentoo
sys-kernel/linux-headers:  6.10::gentoo (virtual/os-headers)
sys-libs/glibc:            2.40::gentoo
Repositories:

gentoo
    location: /var/db/repos/gentoo
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/gentoo.git
    priority: -1000
    volatile: False

CachyOS-kernels
    location: /var/db/repos/CachyOS-kernels
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/CachyOS-kernels.git
    masters: gentoo
    volatile: False

ROKO__
    location: /var/db/repos/ROKO__
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/ROKO__.git
    masters: gentoo
    volatile: False

crossdev
    location: /var/db/repos/crossdev
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/crossdev.git
    masters: gentoo
    volatile: False

dlang
    location: /var/db/repos/dlang
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/dlang.git
    masters: gentoo
    volatile: False

gentoo-unity7
    location: /var/db/repos/gentoo-unity7
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/gentoo-unity7.git
    masters: gentoo
    volatile: False

gentoo-zh
    location: /var/db/repos/gentoo-zh
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/gentoo-zh.git
    masters: gentoo
    volatile: False

guru
    location: /var/db/repos/guru
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/guru.git
    masters: gentoo
    volatile: False

haskell
    location: /var/db/repos/haskell
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/haskell.git
    masters: gentoo
    volatile: False

kde
    location: /var/db/repos/kde
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/kde.git
    masters: gentoo
    volatile: False

lto-overlay
    location: /var/db/repos/lto-overlay
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/lto-overlay.git
    masters: gentoo mv
    volatile: False

mv
    location: /var/db/repos/mv
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/mv.git
    masters: gentoo
    volatile: False

steam-overlay
    location: /var/db/repos/steam-overlay
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/steam-overlay.git
    masters: gentoo
    volatile: False

td5038-overlay
    location: /var/db/repos/td5038-overlay
    sync-type: git
    sync-uri: https://github.com/td5038/ebuilds.git
    masters: gentoo gentoo-unity7
    volatile: False

the-anime-team
    location: /var/db/repos/the-anime-team
    sync-type: git
    sync-uri: https://github.com/an-anime-team/gentoo-ebuilds.git
    masters: gentoo
    volatile: False

wayland-desktop
    location: /var/db/repos/wayland-desktop
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/wayland-desktop.git
    masters: gentoo
    volatile: False

xarblu-overlay
    location: /var/db/repos/xarblu-overlay
    sync-type: git
    sync-uri: https://github.com/gentoo-mirror/xarblu-overlay.git
    masters: gentoo
    volatile: False

Binary Repositories:

gentoobinhost
    priority: 1
    sync-uri: https://distfiles.gentoo.org/releases/amd64/binpackages/23.0/x86-64_llvm

ACCEPT_KEYWORDS="amd64 ~amd64"
ACCEPT_LICENSE="*"
ADDR2LINE="llvm-addr2line"
AR="llvm-ar"
AS="clang -c"
CBUILD="x86_64-pc-linux-gnu"
CC="clang"
CFLAGS="-march=native -O2 -pipe -ggdb"
CHOST="x86_64-pc-linux-gnu"
CONFIG_PROTECT="/etc /usr/share/config /usr/share/gnupg/qualified.txt"
CONFIG_PROTECT_MASK="/etc/ca-certificates.conf /etc/dconf /etc/env.d /etc/fonts/fonts.conf /etc/gconf /etc/gentoo-release /etc/revdep-rebuild /etc/sandbox.d"
CPP="clang-cpp"
CXX="clang++"
CXXFLAGS="-march=native -O2 -pipe -ggdb"
DISTDIR="/var/cache/distfiles"
ENV_UNSET="CARGO_HOME DBUS_SESSION_BUS_ADDRESS DISPLAY GDK_PIXBUF_MODULE_FILE GOBIN GOPATH PERL5LIB PERL5OPT PERLPREFIX PERL_CORE PERL_MB_OPT PERL_MM_OPT XAUTHORITY XDG_CACHE_HOME XDG_CONFIG_HOME XDG_DATA_HOME XDG_RUNTIME_DIR XDG_STATE_HOME"
FCFLAGS="-march=native -O2 -pipe -ggdb"
FEATURES="assume-digests binpkg-docompress binpkg-dostrip binpkg-logs binpkg-multi-instance buildpkg-live config-protect-if-modified distlocks ebuild-locks fixlafiles ipc-sandbox merge-sync merge-wait multilib-strict network-sandbox news parallel-fetch parallel-install pid-sandbox pkgdir-index-trusted preserve-libs protect-owned qa-unresolved-soname-deps sandbox sfperms strict unknown-features-warn unmerge-logs unmerge-orphans userfetch userpriv usersandbox usersync xattr"
FFLAGS="-march=native -O2 -pipe -ggdb"
GENTOO_MIRRORS="http://distfiles.gentoo.org"
LANG="en_US.utf8"
LD="mold"
LDFLAGS="-Wl,-O1 -Wl,--as-needed -Wl,-z,pack-relative-relocs -Wl,--as-needed"
LEX="flex"
MAKEOPTS="-j16 -l16"
NM="llvm-nm"
OBJCOPY="llvm-objcopy"
OBJDUMP="llvm-objdump"
PKGDIR="/var/cache/binpkgs"
PORTAGE_CONFIGROOT="/"
PORTAGE_RSYNC_OPTS="--recursive --links --safe-links --perms --times --omit-dir-times --compress --force --whole-file --delete --stats --human-readable --timeout=180 --exclude=/distfiles --exclude=/local --exclude=/packages --exclude=/.git"
PORTAGE_TMPDIR="/var/tmp"
RANLIB="llvm-ranlib"
READELF="llvm-readelf"
SHELL="/bin/bash"
STRINGS="llvm-strings"
STRIP="llvm-strip"
USE="X a52 aac acl acpi activities alsa amd64 bluetooth branding bzip2 cairo cdda cdr cet clang crypt cups dbus declarative dist-kernel dri dts dvd dvdr encode exif flac flatpak gdbm gif gpm gtk gui iconv icu ipv6 jpeg kde kf6compat kwallet lcms libnotify libtirpc llvm-libunwind lto mad mng mp3 mp4 mpeg multilib ncurses networkmanager nls nvenc ogg opengl openmp pam pango pcre pdf pipewire plasma png policykit ppds pulseaudio qml qt5 qt6 readline screencast sdl seccomp semantic-desktop sound spell ssl startup-notification svg systemd test-rust tiff truetype udev udisks unicode upower usb vaapi vorbis vulkan wayland widgets wxwidgets x264 xattr xcb xft xml xv xvid zeroconf zlib" ABI_X86="64" ADA_TARGET="gcc_12" APACHE2_MODULES="authn_core authz_core socache_shmcb unixd actions alias auth_basic authn_anon authn_dbm authn_file authz_dbm authz_groupfile authz_host authz_owner authz_user autoindex cache cgi cgid dav dav_fs dav_lock deflate dir env expires ext_filter file_cache filter headers include info log_config logio mime mime_magic negotiation rewrite setenvif speling status unique_id userdir usertrack vhost_alias" CALLIGRA_FEATURES="karbon sheets words" COLLECTD_PLUGINS="df interface irq load memory rrdtool swap syslog" CPU_FLAGS_X86="mmx mmxext sse sse2 aes avx avx2 f16c fma3 pclmul popcnt rdrand sha sse3 sse4_1 sse4_2 sse4a ssse3 vpclmulqdq" ELIBC="glibc" GPSD_PROTOCOLS="ashtech aivdm earthmate evermore fv18 garmin garmintxt gpsclock greis isync itrax mtk3301 ntrip navcom oceanserver oncore rtcm104v2 rtcm104v3 sirf skytraq superstar2 tsip tripmate tnt ublox" INPUT_DEVICES="libinput" KERNEL="linux" LCD_DEVICES="bayrad cfontz glk hd44780 lb216 lcdm001 mtxorb text" LLVM_SLOT="18" LUA_SINGLE_TARGET="lua5-1" LUA_TARGETS="lua5-1" OFFICE_IMPLEMENTATION="libreoffice" PHP_TARGETS="php8-2" POSTGRES_TARGETS="postgres15" PYTHON_SINGLE_TARGET="python3_12" PYTHON_TARGETS="python3_12 pypy3" RUBY_TARGETS="ruby31 ruby32" VIDEO_CARDS="amdgpu fbdev intel radeon radeonsi vesa dummy nvidia" XTABLES_ADDONS="quota2 psd pknock lscan length2 ipv4options ipp2p iface geoip fuzzy condition tarpit sysrq proto logmark ipmark dhcpmac delude chaos account" llvm_slot="18" python_targets="pypy3 python3_12" video_cards="amdgpu dummy fbdev intel nvidia radeon radeonsi vesa"
Unset:  ARFLAGS, ASFLAGS, CCLD, CONFIG_SHELL, CPPFLAGS, CTARGET, CXXFILT, ELFEDIT, EMERGE_DEFAULT_OPTS, EXTRA_ECONF, F77FLAGS, FC, GCOV, GPROF, INSTALL_MASK, LC_ALL, LFLAGS, LIBTOOL, LINGUAS, MAKE, MAKEFLAGS, PORTAGE_BINHOST, PORTAGE_BUNZIP2_COMMAND, PORTAGE_COMPRESS, PORTAGE_COMPRESS_FLAGS, PORTAGE_RSYNC_EXTRA_OPTS, PYTHONPATH, RUSTFLAGS, SIZE, YACC, YFLAGS
```

The complete build log will be attached to the issue. Here it is: 
[build.log](https://github.com/user-attachments/files/16642946/build.log)

**Comment by td5038** [comment]
@xarblu

**Comment by xarblu** [comment]
Should be fixed by #507
